### DISCUSSION TITLE
In mathematics, the Gaussian distribution (graph) is better known as a what curve?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#369] [Galaxy Trivia] [answered]
**Hint:** 🔔

**Comment by ghost** [comment]
It is symmetric about the mean and indicates that values near the mean occur more frequently than the values that are farther away from the mean

**Comment by peterbaccaro** [comment]
The 'Bell Curve' is the informal name given to the Gaussian Distribution graph.

**Comment by thuongtruong1009** [comment, marked as answer]
In mathematics, the Gaussian distribution graph is better known as a "bell curve" or "normal curve". It is named after the mathematician Carl Friedrich Gauss, who first described its properties. The bell curve is a continuous probability distribution that is symmetric around the mean and describes the probability distribution of a random variable that is normally distributed. It is widely used in statistics, natural sciences, social sciences, and engineering to model and analyze data that is normally distributed.
